src/corelibs/U2Gui/src/ObjectViewModel.cpp

```diff
@@ -364,222 +377,214 @@ void GObjectViewWindow::setupViewMenu(QMenu *m) {
 /// Utils
 
 GObjectViewWindow *GObjectViewUtils::findViewByName(const QString &name) {
-    QList<MWMDIWindow *> windows = AppContext::getMainWindow()->getMDIManager()->getWindows();
-    QList<GObjectViewWindow *> result;
-    foreach (MWMDIWindow *w, windows) {
-        if (w->windowTitle() == name) {
-            GObjectViewWindow *v = qobject_cast<GObjectViewWindow *>(w);
-            if (v != NULL) {
-                return v;
+    QList<MWMDIWindow *> mdiWindows = AppContext::getMainWindow()->getMDIManager()->getWindows();
+    for (MWMDIWindow *mdiWindow : mdiWindows) {
+        if (mdiWindow->windowTitle() == name) {
+            GObjectViewWindow *objectViewWindow = qobject_cast<GObjectViewWindow *>(mdiWindow);
```

Comment:
`SAFE_POINT(objectViewWindow  != nullptr);`

Reply:
mdiWindows is a QList that can never be nullptr here. I think you mean objectViewWindow pointer.
I would avoid SAFE_POINT here in this review because it will change the current UGENE behavior and this is what I want to avoid.